src/utils/components/CPUMemory/CPUMemory.tsx

```diff
@@ -14,20 +14,21 @@ import { Skeleton } from '@patternfly/react-core';
 import { isRunning } from '@virtualmachines/utils';
 
 type CPUMemoryProps = {
+  fetchVMI?: boolean;
   vm: V1VirtualMachine;
 };
 
-const CPUMemory: FC<CPUMemoryProps> = ({ vm }) => {
+const CPUMemory: FC<CPUMemoryProps> = ({ fetchVMI = true, vm }) => {
   const { t } = useKubevirtTranslation();
   const itMatcher: V1InstancetypeMatcher = vm?.spec?.instancetype;
   const { instanceType, instanceTypeLoaded, instanceTypeLoadError } = useInstanceType(itMatcher);
   const isVMRunning = isRunning(vm);
-  const { vmi, vmiLoadError } = useVMI(getName(vm), getNamespace(vm));
+  const { vmi, vmiLoadError } = useVMI(getName(vm), getNamespace(vm), fetchVMI);
```

Comment:
We can't use hooks conditionally so i added this prop to not fetch vmi when we are displaying template CPU informations